README.md

```diff
@@ -118,6 +134,24 @@ LINK NODE1-NODE2
     NODES NODE1 NODE2
     TARGET prometheus:http:localhost:9090:192.168.178.32:Gi0/1:ifHCInOctets
 ```
+
+This example uases a free-text query to get the throughput of the busiest link in a LAG as a dual query, getting both In and Out:
+```
+LINK NODE1-NODE2
+    NODES NODE1 NODE2
+    TARGET prometheus:http:localhost:9090:max(irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8:max(irate(ifHCInOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8
+```
+In this case we're identifying the interfaces that make up this LAG using a regex query on the ifAlias label.
+
+Modifed to use single query that gets the maximium of both In & Out throughput:
+```
+LINK NODE1-NODE2
+    NODES NODE1 NODE2
+    TARGET max(irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]) or irate(ifHCOutOctets{ifAlias=~".*ae232.*",instance="192.168.178.32"}[10m]))*8
+```
+These free-text PromQL queries can be very powerful.
+
```

Comment:
You could add a friendly warning that depending on the query, it can be quite slow.
Depending on the query, number of queries and the how often you refresh the wmap(s), this can have an impact.
Maybe people that uses the datasource are probably aware of that already and we don't care though. Your call :)

Reply:
Added a quick warning. It can't hurt =)